R/ds.R

```diff
@@ -0,0 +1,115 @@
+# DS
+
+#' Disposition Dataset
+#'
+#' A SDTM DS dataset from the CDISC pilot project
+#'
+#' @source \url{https://bitbucket.cdisc.org/projects/CED/repos/sdtm-adam-pilot-project/browse/updated-pilot-submission-package/900172/m5/datasets/cdiscpilot01/tabulations/sdtm/ds.xpt}
+#' "ds"
+
+library(dplyr)
+library(labelled)
+library(tidyselect)
+library(admiral)
+library(admiral.test)
+
+set.seed(1)
+
+# Reading input data
+data("dm")
+data("suppdm")
+
+data("ds")
```

Comment:
Should be `"admiral_ds"` instead of `"ds"`.

Reply:
updated